sharepoint_connector_config.yml

```diff
@@ -0,0 +1,42 @@
+#Configurations for the SharePoint 2016 Connector
+
+#SharePoint 2016 configuration settings
+#The client id generated while app registration on sharepoint server to access the contents
+sharepoint.client_id: dummy
+#The secret generated while app registration on sharepoint server
+sharepoint.client_secret: dummy
+#The authentication realm of a SharePoint farm
+sharepoint.realm: dummy
+#The address of the sharepoint farm. Example: http://sharepoint-host:14293/
+sharepoint.host_url: http://10.0.9.211:14682
+#Specifies the site collections whose contents the user wants to fetch and index.
+sharepoint.site_collections: 
+    - Connector
+#Elastic WorkPlace configuration settings
+#Access token for Workplace search authentication
+ws.access_token: "52b30c01abe45261fb719d76b67b6520fa429f5a1d1ad8130838adae538982f5"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
```

Comment:
I would expand these configuration properties and avoid using ambiguous acronyms. Example: `workplace_search.access_token`.

Reply:
Done